ui/app/AppLayouts/Wallet/controls/ManageTokensCommunityTag.qml

```diff
@@ -26,7 +31,8 @@ Control {
     }
 
     QtObject {
-        id: d
+        id: priv
```

Comment:
We have an agreement (I cannot find now the doc but you could trust me ;) ) were we are using the naming `d` for the private object that declares private properties in qml (we are applying this rule in the majority of cases).